nbclassic/notebookapp.py

```diff
@@ -274,6 +274,16 @@ def initialize_handlers(self):
                 {"url": self.serverapp.base_url+"files/{0}"}
             )
         )
+        # Add a redirect from /nbclassic to /nbclassic/tree
+        # if both notebook>=7 and nbclassic are installed.
+        if len(nbclassic_path()) > 0:
+            handlers.append(
+                (
+                    rf"/nbclassic",
```

Comment:
Thought about it last night.  For maximum flexibility, this could just be `nbclassic_path()` rather than `rf"/nbclassic"` that way if the path ever changes or becomes a configuration setting, the only place it has to chance is in nbclassic_path().  

Either way works (I just tested) so can go as-is or with that minor tweak.

Reply:
Was going to implement your proposal, but then the redirect url should also be updated, which may be confusing at some point while reading the code. For now, that value is hardcode, so I am leaving as it is for now.

Reply:
> Either way works (I just tested) so can go as-is or with that minor tweak.

Thx again for the tests and time. Really appreciated.

Reply:
Yeah, I can see it being hard to read/mentally parse what is going on with it being the function call both places.

Reply:
@mcrutch The commit and release does not work for me. (it redirects from `/nbclassic` to `/nbclassic/tree`, it should redirect to `/tree`. How is it working for you?

Reply:
So, if you are using NBv6, you should never go to /nbclassic.  If you are using NBv7 and nbclassic, nbclassic is running at /nbclassic/tree and that is expected behavior.  I will poke around a bit when I get some time, but going from /nbclassic to /nbclassic/tree was the intended behavior given that you should only get to /nbclassic when you are using NBv7

Reply:
I have tested again and it works at expected. I will close this comment. Thx.